docs/setup/llm.md

```diff
@@ -0,0 +1,23 @@
+# Large Language Models
+
+!!! warning
+
+    Requires [Ollama](https://github.com/ollama/ollama) to run. This particular setup was tested with [Ollama version 0.1.27](https://ollama.com/download) on an M1 MBP.
+
+[Ollama](https://github.com/ollama/ollama) is used for the unstructured generative component of Privacy Fingerprint. It provides a simple interface to download quantised models and run inference locally.
+
+## Start Ollama
```

Comment:
Do we bluntly say "Install ollama" anywhere?

Reply:
there's a big warning at the top that says it requires Ollama to run - i think this is sufficient